runtime/environment.go

```diff
@@ -0,0 +1,24 @@
+package runtime
+
+import (
+	"cosmossdk.io/core/appmodule"
+	storetypes "cosmossdk.io/store/types"
+)
+
+func NewEnvironment(storeKey *storetypes.KVStoreKey, memKey *storetypes.MemoryStoreKey) appmodule.Environment {
```

Comment:
Why not passing `... store.KVStoreService` directly (and type check with kv service it is)?

Personally, I am not a fan of a constructor that will 90% of the time take a nil argument.